addons/project/static/src/views/project_task_kanban/project_task_kanban_compiler.js

```diff
@@ -1,70 +0,0 @@
-/** @odoo-module **/
-
-import { KanbanCompiler } from "@web/views/kanban/kanban_compiler";
-import { append, createElement } from "@web/core/utils/xml";
-
-export class ProjectTaskKanbanCompiler extends KanbanCompiler {
-    setup() {
-        super.setup();
-        this.subtaskListComponentCompiled = {
-            button: false,
-            component: false,
-        };
-        this.compilers.push(
-            { selector: ".subtask_list_button", fn: this.compileSubtaskListButton },
-            { selector: "div.kanban_bottom_subtasks_section", fn: this.compileSubtaskListComponent },
-        );
-    }
-
-    /**
-     * @param {Element} el
-     * @returns {Element}
-     */
-    compileSubtaskListButton(el) {
-        this.subtaskListComponentCompiled.button = true;
-        el.setAttribute("t-on-click", `() => __comp__.state.folded = !__comp__.state.folded`);
-        const compiled = createElement(el.nodeName);
-        for (const { name, value } of el.attributes) {
-            compiled.setAttribute(name, value);
-        }
-
-        for (const child of el.childNodes) {
-            append(compiled, this.compileNode(child));
-        }
-
-        return compiled;
-    }
-
-    /**
-     * @param {Element} el
-     * @returns {Element}
-     */
-    compileSubtaskListComponent(el) {
-        this.subtaskListComponentCompiled.component = true;
-        el.setAttribute("t-if", `!__comp__.state.folded and !selection_mode`);
-        const compiled = createElement(el.nodeName);
-        for (const { name, value } of el.attributes) {
-            compiled.setAttribute(name, value);
-        }
-        const listContainer = createElement('widget');
-        const listElemenent = createElement('SubtaskKanbanList');
-        listElemenent.setAttribute("record", '__comp__.props.record');
-
-        append(listContainer, listElemenent);
-        append(compiled, listContainer);
-
-        return compiled;
-    }
-
-    /**
-     * @override
-     */
-    compile(key, params = {}) {
-        const newRoot = super.compile(key, params);
-        if (this.subtaskListComponentCompiled.component !== this.subtaskListComponentCompiled.button) {
-            // Error since one of them is not compiled
-            throw new Error("The subtask list component cannot be rendered if the button and the component are not in the view definition.");
```

Comment:
Is it expected to completely remove that?

I am not against but we no longer check if the both widgets will be rendered in the kanban view. 🤔

Reply:
Mmh yep, i'll add the compiler back,
I had to do a few modifications to the way we add the widgets compilers to the compilers array, we need them at the start so they are before the default *widget* selector,
maybe it's better to just check the presence of the component in the compile method, tell me what you think

Reply:
Yes we could, the idea is to be sure the component SubtaskList can be correctly mounted. 🙂

Reply:
Hello, so what do you think about this? is this verification ok?
https://github.com/odoo/odoo/pull/126692/files#diff-8a9958556e93463c737373caf40c9e24b57a3638f30f9a18b79870239bd141da

Reply:
Oh nice, good idea. 🤩